### PR TITLE
this is an Article about event loop by Amish Khan

### PR DESCRIPTION
**Pull request by AMISH-KHAN**
This is an article explaining event loop and how Javascript is able to perform asynchronous task while being a single threaded language

### CODE REVIEW

File: Articles/Web Development/Exploring_theBlackBox/articlemap.xml
Comment (brianpereboom) [inline Articles/Web Development/Exploring_theBlackBox/articlemap.xml:1]: Please add `<url>
  <loc>Web Development/Exploring_theBlackBox</loc>
  <image>images/Javascript-thumbnail.png</image>
</url>` to the articlemap.xml file in the root directory and delete this file.

File: Articles/Web Development/Exploring_theBlackBox/images/AmishLInkedIn.png
Comment (brianpereboom) [inline Articles/Web Development/Exploring_theBlackBox/images/AmishLInkedIn.png:1]: Is this image used in the article? I recommend writing a biography about yourself in the `Meet Our Creators` directory. You may include this image there.

**Review by brianpereboom — CHANGES_REQUESTED** [review]
Your article looks good, but I will need a couple changes before I can approve the request. Please read my comments and make the necessary changes. Thank you.

File: Articles/Web Development/Exploring_theBlackBox/images/AmishLInkedIn.png
Comment (AMISH-KHAN) [reply Articles/Web Development/Exploring_theBlackBox/images/AmishLInkedIn.png:1]: I thought of including the image as a logo in the article that will redirect to my linkedIn profile but drop that idea.
I will write a biography soon

Comment (brianpereboom) [reply Articles/Web Development/Exploring_theBlackBox/images/AmishLInkedIn.png:1]: The thumbnail for the article will be displayed on the home page and direct the user to the article.

**Review by brianpereboom — APPROVED** [review]

File: Articles/Web Development/Exploring_theBlackBox/articlemap.xml
Comment (AMISH-KHAN) [reply Articles/Web Development/Exploring_theBlackBox/articlemap.xml:1]: I have made the required changes is there anything else I can do?

**Comment by AMISH-KHAN** [comment]
when this website will be live??

**Comment by brianpereboom** [comment]
It should be within 10 minutes.

**Pull request merged**